packages/dev/src/examples/index.ts

```diff
@@ -17,12 +19,19 @@ export type ExampleItem = {
 
 export const examplesFlat: ExampleItem[] = imports.keys().map(key => {
   const module = imports(key)
-  return {
-    title: module.title,
-    subTitle: module.subTitle,
-    category: module.category,
-    component: module.component,
+  const category = key.match(/\.\/[\w-]+\/([\w-]+)\//)?.[1] as string
+
+  const exampleItem = {
+    ...module,
+    category: kebabToTitleCase(category),
+  }
+
+  if (module.default) {
```

Comment:
While this approach works, I'm not sure whether it's a good pattern. What if we exported transition components as named exports with a certain name? For example `module.transitionComponent`. It will allow us to introduce other component types in the future if we need them.

Reply:
Good point! I've changed default to `transitionComponent`.